posts/bayesian-model-comparison/index.qmd

```diff
@@ -291,7 +299,7 @@ Here we will say that a difference of 1% in performance between the two models m
 #| code-fold: true
 #| code-summary: code for `get_t_dist_posterior_probs`
 def get_t_dist_posterior_probs(
```

Comment:
These names, we happy with them or better to have them complete, i.e. `get_t_distribution_posterior_probabilities`?